save-backend/src/main/kotlin/com/saveourtool/save/backend/service/LnkUserOrganizationService.kt

```diff
@@ -196,10 +197,20 @@ class LnkUserOrganizationService(
         }
         .map { it.organization }
 
+    /**
+     * @param user
+     * @param filters
+     * @return [Organization]s that are connected to the [user] anf matching filters
+     */
+    fun getOrganizationsAndRolesByUserAndFilters(user: User, filters: OrganizationFilters): List<LnkUserOrganization> =
+            lnkUserOrganizationRepository.findByUserId(user.requiredId()).filter { lnkUserOrganization ->
+                lnkUserOrganization.organization.let { it.name.contains(filters.prefix) && it.status in filters.statuses }
```

Comment:
`contains(filters.prefix)` or `startsWith(filters.prefix)`?

Reply:
In my opinion, it is better to make ```contains```, but if it needs to be redone later, I will do it.

Reply:
Then why is it called to be prefix?

Reply:
It makes sense... Redid with ```startsWith```